### PR TITLE
Post-news-loader

### PR DESCRIPTION
**Pull request by RuslanHrybchuk**

### CODE REVIEW

File: src/app/component/about-page/about-page.component.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
+<div class="container">
```
Comment (nabermi2020) [inline src/app/component/about-page/about-page.component.html]: I guess it's not related to this ticket, please revert it

File: src/app/component/post-news-loader/post-news-loader.component.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <div class="img"></div>
+  <span class="text-1">{{ 'post-news-loader.text-1' | translate }}</span>
```
Comment (nabermi2020) [inline src/app/component/post-news-loader/post-news-loader.component.html]: replace magic numbers

File: src/app/component/about-page/about-page.component.html (hunk 1 above)
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: fixed

File: src/app/component/post-news-loader/post-news-loader.component.html (hunk 2 above)
Comment (RuslanHrybchuk) [reply src/app/component/post-news-loader/post-news-loader.component.html]: fixed

**Comment by nabermi2020** [comment]
checked

**Pull request merged**